packages/ng-primitives/schematics/input-form-field/files/__name__.component.html.template

```diff
@@ -0,0 +1,5 @@
+<div ngpFormField>
+  <label ngpLabel>label</label>
+  <p ngpDescription>description</p>
+  <input ngpInput placeholder="placeholder" />
+</div>
```

Comment:
I'm not sure how useful an `input-form-field` component would be in general to have in an application. There are too many things someone may want to customize for each form field for it to be abstracted into one component.

For example, if I use this generator to create a component, and I go to build a form using it, e.g:

```html
<form>

<!-- Email -->
<app-input-form-field></app-input-form-field>

<!-- Name -->
<app-input-form-field></app-input-form-field>

</form>
```

How would I define the label text, the placeholder text, the input field type attribute, the description text, any errors, connect Angular forms etc.. There are too many complexities to make this useful. 

I think a better approach is to continue the way you have been with button and input, where we generate a component for each individual primitive, e.g. form-field, label, description etc.. That way they can be pieced together however they need, but it still speeds up the setup for someone adding them to their application 😊

Reply:
I think the `input-form-field` generator is useless. What we need rather is a `form-field` generator right ?

Reply:
Exactly 👍

Reply:
Done

Reply:
Thank you!